### PR TITLE
Truncated description on game cards and decreased size of description on game page

### LINKED ISSUE #258: Fix the way that game descriptions are displayed
As of right now, game descriptions make the cards on the game browse page way too long. We need to display a shortened version of the description. There are also several XML special characters like `&#10;` that we need to interpret correctly.

Image

### PR DESCRIPTION
**Pull request by willcravitz**
This code does 2 things to address issue #258:
1. Truncates the description to **20 words** on game cards on the game grid page 
2. On the game detail page, decreases the size of the description (from `h3` to `p`) and moves it into the information section

### CODE REVIEW

File: src/templates/games/game_detail.html
Comment (danielngira) [inline src/templates/games/game_detail.html:37]: The changes in this file reflect the goal of the issue successfully. You changed the content of the descriptions from a h3 to a paragraph. You've also moved the description to the information section. Everything seems okay here. I just have a small suggestion on the style. Since we've used h1 for the game title, would it be better to use h2 for the attributes of the game instead of h3? For instance, having the title "Description as h2

File: src/templates/games/game_grid.html
Comment (danielngira) [inline src/templates/games/game_grid.html:57]: image

The description in the game grid looks perfect! I was thinking, is it possible to let the user know/sth that can signal to them that they can read more about the game description so they can know the game description was purposely truncated and there's a complete version of it? Can a "read more" under the description be a useful indicator or would it be redundant since the whole grid itself is a link to the game detail page?

**Review by danielngira — COMMENTED** [review]
The code seems correct and achieves the intended purpose. I left two comments, one is a minor one on style and the other one was me contemplating how to handle the description on the grid view page. The changes I suggested are optional really so I'll approve the PR once you contemplate them

File: src/templates/games/game_detail.html
Comment (willcravitz) [reply src/templates/games/game_detail.html:37]: I think this suggestion makes a lot of sense, but when I initially made this change I decided (very subjectively) that `h3` looked better. I think I'm going to keep it for now but would have no problem with someone opening up a new issue to adjust game detail styling in the future.

File: src/templates/games/game_grid.html
Comment (willcravitz) [reply src/templates/games/game_grid.html:57]: Similar to what I said in the comment above, I think this is a really good suggestion, but I'd rather leave it up to future team members to open up a new game card styling issue. The main focus of this issue was to make sure that descriptions don't make the cards too big.

File: src/templates/games/game_detail.html
Comment (danielngira) [reply src/templates/games/game_detail.html:37]: I understand. Thank you for letting me know!

File: src/templates/games/game_grid.html
Comment (danielngira) [reply src/templates/games/game_grid.html:57]: Like with the previous comment, I think your judgement makes sense. I can resolve these comments now

**Review by danielngira — APPROVED** [review]

**Review by majorsylvie — APPROVED** [review]
Looks great! The Grid view is significantly cleaner and more organized. Thank you for adding the `...` to signal that there is more to read. 

Similarly the detail view feels even more put together now that the size has been meaningfully decreased. 

This code is good to merge. I do have two further suggestions on more changes to make!

I would recommend changing the alignment of detail view elements to address the problem of (particularly large vertical) images creating whitespace above the description:
image



I agree that a great next step would be adding a `Read More` button / link to expand a given card.

An idea for that is to have it bring up the card in a "center peek" view. Notion does this for example:
image

**Comment by willcravitz** [comment]
Closing comments: The descriptions on game cards are now limited to 20 words followed by an ellipses. Descriptions on the detail page took up too much space as a header, so they are now paragraph text.

@danielngira and @majorsylvie both suggested ways to further improve the user experience on both the game grid and detail pages which should be addressed in future issues.

**Comment by majorsylvie** [comment]
Issue Score: Excellent

Comments:
Thank you @danielngira for the good review and offering suggestions
@willcravitz great work! Appreciate the good closing comment as well as the list of things done by this PR!

Keep moving forward!

**Comment by ksigvaldason** [comment]
closes #422

**Pull request merged**
